ml-agents/mlagents/trainers/coma/trainer.py

```diff
@@ -58,7 +58,7 @@ def __init__(
             PPOSettings, self.trainer_settings.hyperparameters
         )
         self.seed = seed
-        self.policy: Policy = None  # type: ignore
+        self.policy: TorchPolicy = None  # type: ignore
```

Comment:
I guess this still throws a mypy error?

Reply:
The `# type: ignore` is to handle it being assigned None - otherwise you'd have to use `Optional[TorchPolicy]` and check that it was non-None everywhere you used it.